tests/bluetooth/audio/ccid/src/main.c

```diff
@@ -0,0 +1,96 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <errno.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <sys/types.h>
+
+#include <zephyr/bluetooth/audio/ccid.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/fff.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/sys/util_macro.h>
+
+#include <zephyr/ztest_test.h>
+#include <zephyr/ztest_assert.h>
+
+DEFINE_FFF_GLOBALS;
+
+ZTEST_SUITE(audio_ccid_test_suite, NULL, NULL, NULL, NULL, NULL);
+
+#define MAX_CCID_CNT 256
+
+static ZTEST(audio_ccid_test_suite, test_bt_ccid_alloc_value)
+{
+	const int ret = bt_ccid_alloc_value();
+
+	zassert_true(ret >= 0 && ret <= UINT8_MAX, "Unexpected return value %d", ret);
+}
+
+static ZTEST(audio_ccid_test_suite, test_bt_ccid_alloc_value_more_than_max)
+{
+	/* Verify that we can allocate more than max CCID if they are no registered */
```

Comment:
```suggestion
	/* Verify that we can allocate more than max CCID if there are no registered */
```

Reply:
`they` (the CCIDs) is correct here I would say